ccd-definition/CaseField/CaseFieldGAspec.json

```diff
@@ -134,7 +134,29 @@
   },
   {
     "CaseTypeID": "CIVIL",
-    "ID": "gaDetailsRespondentSol",
+    "ID": "gaDetailsMasterCollection",
+    "Label": " ",
+    "ElementLabel": "Application Type",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "GeneralApplicationDetailsGAspec",
+    "SecurityClassification": "Public",
+    "_Definition":  "The details of the child application on the case for Judge",
+    "_Category":  "COMMON"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "respondentSolGaAppDetails",
+    "Label": " ",
+    "ElementLabel": "Application Type",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "GeneralApplicationDetailsGAspec",
+    "SecurityClassification": "Public",
+    "_Definition":  "The details of the child application on the case",
```

Comment:
should we give all the definition proper description similar to above gadetailsmaster collection ?

Reply:
Modified the definition of claimantgaDetails , respondentSolGaDetails and respondentSolTwoGaAppDetails  in casefields.json